### ISSUE TITLE
Admin static files is not serving out of box

### ISSUE THREAD

**Issue by matacoder** [lemonsaurus/django-starter#3]
/admin/login/?next=/admin/ give us 404 errors:
django-starter | Not Found: /static/admin/css/fonts.css
django-starter | Not Found: /static/admin/css/nav_sidebar.css
django-starter | Not Found: /static/admin/css/base.css
django-starter | Not Found: /static/admin/css/responsive.css
django-starter | Not Found: /static/admin/js/nav_sidebar.js
django-starter | Not Found: /static/admin/css/login.css

**Comment by lemonsaurus** [comment]
Yes, I didn't get this far yet, but this should be resolved. Thanks for making an issue.
